### PR TITLE
Added SmallRye areas in autoissuelabeler.yml

### PR DESCRIPTION
**Pull request by phillip-kruger**
Closing #11018 and opening this one in it's place.

New labels needed

    area/tracing
    area/metrics
    area/health
    area/fault-tolerance

### CODE REVIEW

File: .github/autoissuelabeler.yml [hunk 1]
```diff
@@ -51,12 +51,41 @@
...
   title: "(?i).*graphql.*"
-  notify: [phillip-kruger,jmartisk]  
+  description: "(?i).*graphql.*"
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/tracing, area/smallrye]
+  title: "(?i).*(trace|tracing|telemetry).*"
+  description: "(?i).*(trace|tracing|telemetry).*"
+  notify: [kenfinnigan,Ladicek]
+- labels: [area/config, area/smallrye]
+  title: "(?i).*config.*"
```
Comment (gsmet) [inline .github/autoissuelabeler.yml:67]: OK so I maintain we need that one removed because it will match all bugs using the default template.

Comment (phillip-kruger) [reply .github/autoissuelabeler.yml]: removed.

**Comment by phillip-kruger** [comment]
@gsmet - done and squashed.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Cool, thanks!

**Pull request merged**